File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/viewModels/UpdateEmployeeViewModel.java
package com.unosquare.admin_core.back_end.viewModels;
import lombok.Data;

@Data
public class UpdateEmployeeViewModel {
    private int countryId;
    private String email;
    private int employeeId;
    private int employeeRoleId;
    private int employeeStatusId;
    private String firstName;
    private String lastName;
    private int totalHolidays;


    public UpdateEmployeeViewModel(){

    }

    public UpdateEmployeeViewModel(int countryId, String email, int employeeId, int employeeRoleId, int employeeStatusId,
                                   String firstName, String lastName , int totalHolidays){

        this.countryId = countryId;
        this.email = email;
        this.employeeId = employeeId;
        this.employeeRoleId = employeeRoleId;
        this.employeeStatusId = employeeStatusId;
        this.firstName = firstName;
        this.lastName = lastName;
        this.totalHolidays = totalHolidays;
    }

}

File: v2/AdminCore.WebApp/Models/Employee/UpdateEmployeeViewModel.cs
using System;

namespace AdminCore.WebApi.Models.Employee
{
  public class UpdateEmployeeViewModel
  {
    public int CountryId { get; set; }

    public string Email { get; set; }

    public int EmployeeId { get; set; }

    public int EmployeeRoleId { get; set; }

    public int EmployeeStatusId { get; set; }

    public string Forename { get; set; }

    public DateTime StartDate { get; set; }

    public string Surname { get; set; }
  }
}